### PR TITLE
SIGSRE-65: Add SLO Bootstrap Guide

### PR DESCRIPTION
**Pull request by lisa**
There is still a link to be filled in for the SLO Reporting content, but
it's a work in progress still. The link now points to a Google Doc
(which is not likely to be available to folks outside Red Hat) where
ideation is being fleshed out.

Signed-off-by: Lisa Seelye <lisa@users.noreply.github.com>

### CODE REVIEW

File: slo_bootstrap_guide.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# SLO Bootstrap Guide
+
+Red Hat's SIG-SRE group has created a set of documents which share how some teams engage with the practice of SLOs inside Red Hat. It is our hope that these supporting documents can assist others. Before we get started, there are some [definitions](#step-1-familiarise-definitions) to ensure common understanding of terms and phrases that are used throughout the package.
```
Comment (jeremyeder) [inline slo_bootstrap_guide.md]: Add a sentence here "This guide is meant to be a self-contained bootstrapping guide for teams looking to use the site reliability engineering approach to supporting services."

**Review by mh21 — APPROVED** [review]
LGTM!

**Pull request merged**